package-system/AWSGameLiftServerSDK/build_package_image.py

```diff
@@ -152,8 +152,8 @@ def download_gamelift_server_sdk(working_directory: WorkingDirectoryInfo) -> Non
 def get_custom_build_env():
     if PACKAGE_PLATFORM in (PACKAGE_PLATFORM_OPTIONS[1], PACKAGE_PLATFORM_OPTIONS[2]):
         custom_env = os.environ.copy()
-        custom_env["CC"] = "clang"
-        custom_env["CXX"] = "clang++"
+        custom_env["CC"] = "gcc"
```

Comment:
is `gcc` guaranteed to be present on systems where this script will run? And if not, will an error that indicates its absence be raised? (sorry if this is an obvious question, I am not super familiar with executing this project)

Reply:
Yep! `gcc` comes by default in both our supported ubuntu versions (20.04 LTS and 22.04 LTS)